### DISCUSSION TITLE
Show columns with nulls when API response has array with no data

### DISCUSSION THREAD

**Discussion by sollymay** [grafana/grafana-infinity-datasource#1025] [Q&A]
I have been working on a dashboard that merged many API requests with the same format and have stumbled upon something I can't figure out.

For this API request I normally get an array with objects inside that I can easily query and get the information I need. I then merge these to get the expected result.

The problem I'm facing is that I would like to get None or Null if the field does not exist or if there is no data on that array for each computer field.

I have tried to do this on the computed columns by using an OR to basically pass the field value if it exists or None if it doesn't but that doesn't work.

Basically I just want to compute values irrespective if they exist or not to display None when they don't exist in each individual column name, vs getting No data. Is there a way to do this??

![PXL_20241021_160237811.jpg](https://github.com/user-attachments/assets/388a65bb-4850-4f49-8848-0f7a901f5c93)

Sorry for the janky photo.
